optaplanner-quickstarts/spring-boot-school-timetabling/build.gradle

```diff
@@ -0,0 +1,34 @@
+plugins {
+    id "org.springframework.boot" version "2.2.6.RELEASE"
+    id "io.spring.dependency-management" version "1.0.9.RELEASE"
+    id "java"
+}
+
+group = "org.optaplanner.quickstarts"
+version = "7.38.0-SNAPSHOT"
```

Comment:
Needs to change during releases...

Reply:
Is Gradle necessary? I'm not particularly excited to have to maintain and test everything with two different build systems.
And I'm sure QE wouldn't be either.

Reply:
20% of the java ecosystem use Gradle.
We can't keep ignoring them I think. Definitly worth a discussion.